src/Microsoft.SqlTools.Credentials/Microsoft.SqlTools.Credentials.csproj

```diff
@@ -20,10 +20,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="System.Data.SqlClient" Version="4.5.0-preview2-26406-04" />
-    <PackageReference Include="Microsoft.SqlServer.Management.SmoMetadataProvider" Version="$(SmoPackageVersion)" />
-    <PackageReference Include="Microsoft.SqlServer.Management.SqlScriptPublishModel" Version="$(SmoPackageVersion)" />
-    <PackageReference Include="Microsoft.SqlServer.SqlParser" Version="$(SmoPackageVersion)" />
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
```

Comment:
Academic question: This is a little unexpected removal I would think. What I am wondering at is the following. Does the code in this project depend directly on Newtonsoft.json or did it have this dependency due to one of the SmoMetadataProvider, SqlScriptPublishModel, and SqlParser codebases.

Reply:
looks like linux FileTokenStorage needs it. I think I just selected one too many lines before I hit delete.